### PR TITLE
[fix] CI, fixing check-label-preview IF clause Scope

### PR DESCRIPTION
**Pull request by phoinixgrr**
Rationale: https://mattermost.atlassian.net/browse/CLD-5750

### CODE REVIEW

File: .github/workflows/check-label.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ on:
 jobs:
   check-label-preview-env:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'preview-environment')
     steps:
     - name: check-label
```
Comment (jupenur) [inline .github/workflows/check-label.yml:12]: Having this here makes very little sense now that the actual check is higher up. Maybe just remove it altogether?

Comment (phoinixgrr) [reply .github/workflows/check-label.yml:12]: @jupenur  In which actual check you are referring to?

Comment (jupenur) [reply .github/workflows/check-label.yml:12]: The `if` you moved away from under this step. The step now does nothing.

Comment (phoinixgrr) [reply .github/workflows/check-label.yml:12]: @jupenur  The step prints in STD OUT: "label exist, will trigger the preview environment"

Comment (phoinixgrr) [reply .github/workflows/check-label.yml:12]: Is that nothing? 
There is some verbosity introduced by the original author. Is there a reason for removing this?

Comment (jupenur) [reply .github/workflows/check-label.yml:12]: Well yes, but that doesn't seem useful. It does not "check the label" like the name suggests, and the job running should already tell anyone reading the logs that the label exists and the preview environment is getting triggered.

So what I'm saying is this step contains zero functionality: it just logs a line of text, and even that log entry is entirely redundant.

Comment (phoinixgrr) [reply .github/workflows/check-label.yml]: Rebased and pushed the changes asked.

**Review by jupenur — APPROVED** [review]
:shipit:

**Review by saturninoabril — APPROVED** [review]
Thanks @phoinixgrr!

**Pull request merged**